src/command/KeyBindingManager.js

```diff
@@ -410,6 +411,13 @@ define(function (require, exports, module) {
             });
         }
         
+        // Also make sure that the second word of PageUp/PageDown has the first letter in upper case.
+        if (/^Page/.test(key)) {
+            key = key.replace(/(up|down)$/, function (match, p1) {
+                return p1.charAt(0).toUpperCase() + p1.substr(1);
+            });
+        }
```

Comment:
The `if` block above this already takes care of the first letter of all key names and here we're only taking care of the two key names that have the second word.